lyrics-core/src/main/java/com/flipkart/lyrics/specs/ClassName.java

```diff
@@ -0,0 +1,246 @@
+/*
+ * Copyright (C) 2014 Google, Inc.
```

Comment:
is this coming class coming from google? If so, can we use the artifact that contains this class?

Reply:
Cannot find any artifact with this class. But square guys are licensing this class to Google.